specification/Commented_Example.nef

```diff
@@ -302,81 +499,177 @@ INSERT PULSE PROGRAM HERE
+         W     4       CYS   CA     41.2    0      .     .
+         W     4       CYS   CB     27.3    0      .     .
+         W     4       CYS   H      7.9     0      .     .
+         W     4       CYS   HA     4.8     0      .     2
+         W     4       CYS   HB2    2.6     0      .     2
+         W     4       CYS   HB3    2.5     0      .     2
+         W     5       LYS   CA     41.2    0      .     .
+         W     5       LYS   CB     27.3    0      .     .
+         W     5       LYS   CG     23.2    0      .     .
+         W     5       LYS   CD     22.3    0      .     .
+         W     5       LYS   CE     26.3    0      .     .
+         W     5       LYS   H      8.4     0      .     .
+         W     5       LYS   HA     4.2     0      .     .
+         W     5       LYS   HB2    2.2     0      .     .
+         W     5       LYS   HB3    2.1     0      .     .
+         W     5       LYS   HG%    1.4     0      .     .
+         W     5       LYS   HD2    1.6     0      .     .
+         W     5       LYS   HD3    1.5     0      .     .
+         W     5       LYS   HE%    3.1     0      .     .
+         W     6       CA    CA     1.1     0      CA    .
+         W     7       CD    CD     -150.0  0      CD    111
+         Z     @5      GLX   H      9.1     0      .     .
+         Z     @5      GLX   N      118.3   0      .     .
       stop_
 
 # The atom_name by itself serves to distinguish between real atoms ('HA'),
 #  pseudoatoms ('QA'), sets of atoms ('HB%'), sterospecifically assigned atoms
-# ('HB2'), and non-stereospecifically assigned atoms ('HBX').
+# ('HB2'), and non-stereospecifically assigned atoms ('HBx').
 # In the example, Phe 18 HB2/HB3 are stereospecifically assigned, while Leu 19
 # is not.
 # Gly 15 QA is a pseudoatom, i.e. an atom at the HA2/HA3 centroid with zero van
 # der Waals radius.
 # Val 17 HG%/CG% show two methyl groups that overlap in both carbon and proton
 # dimensions.
-# 19 LEU HDX% is the methyl bound to CDX (and not to CDY).
-# The last two shifts (with chain_code @2) are unassigned but observed resonances,
+#
+# 19 LEU HDx% is the methyl bound to CDx (and not to CDy).
+#
+# Chain W illustrates the element and isotope specificatoin rules
+# It is supposed to represent a Ca,Cd peptide complex with 2H la
+         W     2       CYS   HA     4.3     0      .     2
+         W     2       CYS   HB2    2.8     0      .     2
+         W     2       CYS   HB3    2.7     0      .     2beled CYS in 50% H2O/D2O.
+# Residue 6 is Calcium, residue 7 is Cadmium.
+# Some shifts that can not be given in this list because of name clashes are in
+# shiftlist 2 instead.
+# Note that e.g. W  4  CYS H and HA must be referred to as H and HA throughout the file.
+# Whether the atoms in question are in fact 1H or 2H, e.g. in a peak assignment, must
+# be decided by reference to the experiment dimension nucleus.
+#
+# The last two shifts (with chain_code Z) are unassigned but observed resonances,
```

Comment:
Is 'Z' reserved chain code for unassigned but observed  resonances?  'Z' could also used as one of the regular chain code right?!